### ISSUE TITLE
Cannot build container anymore

### ISSUE THREAD

**Issue by dcesari** [ARPA-SIMC/arkiweb-docker-image#2]
I know this package is a headache but...

While trying to rebuild the arkiweb container on a CentOS7 distro the following set of commands:
```
    yum -y copr enable simc/stable
    yum -y copr enable simc/arkiweb
    yum install -y arkiweb httpd
```
fails with the error:
```
...
--> Running transaction check
---> Package arkimet.x86_64 0:1.14-2 will be installed
--> Processing Dependency: libdballe6 >= 8 for package: arkimet-1.14-2.x86_64
Package libdballe6 is obsoleted by libdballe9, but obsoleting package does not provide for requirements
--> Processing Dependency: libdballe.so.8()(64bit) for package: arkimet-1.14-2.x86_64
Package libdballe6 is obsoleted by libdballe9, but obsoleting package does not provide for requirements
---> Package arkiweb.x86_64 0:0.27-6.el7 will be installed
--> Processing Dependency: libdballe.so.8()(64bit) for package: arkiweb-0.27-6.el7.x86_64
Package libdballe6 is obsoleted by libdballe9, but obsoleting package does not provide for requirements
...
--> Finished Dependency Resolution
Error: Package: arkimet-1.14-2.x86_64 (copr:copr.fedorainfracloud.org:simc:arkiweb)
           Requires: libdballe6 >= 8
           Available: libdballe6-8.20-2.el7.x86_64 (copr:copr.fedorainfracloud.org:simc:stable)
               libdballe6 = 8.20-2.el7
Error: Package: arkimet-1.14-2.x86_64 (copr:copr.fedorainfracloud.org:simc:arkiweb)
           Requires: libdballe.so.8()(64bit)
           Available: libdballe6-8.20-2.el7.x86_64 (copr:copr.fedorainfracloud.org:simc:stable)
               libdballe.so.8()(64bit)
Error: Package: arkiweb-0.27-6.el7.x86_64 (copr:copr.fedorainfracloud.org:simc:arkiweb)
           Requires: libdballe.so.8()(64bit)
           Available: libdballe6-8.20-2.el7.x86_64 (copr:copr.fedorainfracloud.org:simc:stable)
               libdballe.so.8()(64bit)
```
I guess some packages aren't available anymore in the copr repos, is there a way to survive?
I still have a binary singularity container working, but as long as we need arkiweb it would be nice to have the chance to rebuild it.

**Comment by edigiacomo** [comment]
You can always download the image from docker hub https://hub.docker.com/repository/docker/arpaesimc/arkiweb.

If you need to rebuild it, I think that a solution could be to build libdballe6 in `simc/arkiweb` repo. I can trigger the build if you want.

**Comment by edigiacomo** [comment]
I've triggered the build, so that we can rebuild the image regardless of Docker Hub availability.

**Comment by edigiacomo** [comment]
`simc/arkiweb` repository is now self-contained (f2d7aa9), please close the issue if everything works fine.

**Comment by dcesari** [comment]
Great, thanks! Everything works now.
